pipelines/e2e/README.md

```diff
@@ -18,6 +18,9 @@ affected by RHTAP services or which results could affect the RHTAP workflow.
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+## Changes in 0.4.1
+* The cleanup-workspace task is now passed a pipelineRunUid parameter to cleanup InternalRequests
```

Comment:
Isn't this wording weird? Shouldn't be something like `The cleanup-workspace task receives now a...`. Honest question.

Reply:
I think it is about the same, but I updated to your suggestion in case that's clearer